src/stcal/jump/twopoint_difference.py

```diff
@@ -122,7 +147,8 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
         # compute 'ratio' for each group. this is the value that will be
         # compared to 'threshold' to classify jumps. subtract the median of
         # first_diffs from first_diffs, take the abs. value and divide by sigma.
-        ratio = np.abs(first_diffs - median_diffs[np.newaxis, :, :]) / sigma[np.newaxis, :, :]
+        dn_jump = first_diffs - median_diffs[np.newaxis, :, :]
```

Comment:
At this point in the flow aren't the pixel values in units of electrons, hence the name `dn_jump` may be misleading?

Reply:
renamed to `e_jump`.